docs/resources/mws_workspaces.md

```diff
@@ -300,26 +300,29 @@ output "databricks_token" {
 }
 ```
 
+In order to create a [Databricks Workspace that leverages GCP Private Service Connect](https://docs.gcp.databricks.com/administration-guide/cloud-configurations/gcp/private-service-connect.html) please ensure that you have read and understood the [Enable Private Service Connect](https://docs.gcp.databricks.com/administration-guide/cloud-configurations/gcp/private-service-connect.html) documentation and then customise the example above with the relevant examples from [mws_vpc_endpoint](mws_vpc_endpoint.md), [mws_private_access_settings](mws_private_access_settings.md) and [mws_networks](mws_networks.md).
```

Comment:
This doc block should be added to "Creating a Databricks on GCP workspace" section instead of "Creating a Databricks on GCP workspace with Databricks-Managed VPC" section.

Reply:
Ah, thanks!